src/modules/utils/index.ts

```diff
@@ -12,4 +15,62 @@ export class UtilsModule extends BaseModule {
 
     return header.substring(7);
   }
+
+  // Token Gating function validates user ownership of wallet + NFT
+  public async validateTokenOwnership(
+    didToken: string,
+    contractAddress: string,
+    contractType: 'ERC721' | 'ERC1155',
+    web3: Web3,
```

Comment:
Curious, why web3 needs to be a param in this case?

Reply:
Just a thought, maybe web3 can be a peer dependency, which can keep our admin sdk light-weight and thin

Reply:
I made it a param to make the mocks more straightforward in the unit tests, and also to allow the users to pass in web3 instances that may have custom options.

I'm not familiar with peer dependencies, looking into it

Reply:
moved into peer dependencies @Ethella